### PR TITLE
set extension defaults with custom format

### PR DESCRIPTION
**Pull request by edavidaja**
rearranged the content into a custom format `posit-docs`

sample: https://connect.posit.it/drafts/posit-doc-theme/

### CODE REVIEW

File: _extensions/posit-docs/_extension.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        right:
+          - icon: book
+            href: https://docs.posit.co
+          - icon: question-circle-fill
+            href: https://support.posit.co/hc/en-us
+          - icon: lightbulb-fill
+            href: https://solutions.posit.co
+  formats:
+    html:
+      theme: [cosmo, theme.scss]
```
Comment (AshleyHenry15) [inline _extensions/posit-docs/_extension.yml:38]: Should we remove cosmo from here since we don't want that added at all?

Comment (edavidaja) [reply _extensions/posit-docs/_extension.yml:38]: Yep, I thought you said it was cosmo based somewhere else but I will pull that out.

Comment (edavidaja) [inline _extensions/posit-docs/_extension.yml:38]:
```suggestion
      theme: [theme.scss]
```

**Review by AshleyHenry15 — APPROVED** [review]

**Comment by AshleyHenry15** [comment]
My testing steps:
- Pulled branch locally
- Created a new quarto website project locally (Desktop dir)
- From terminal, cd into new quarto project
- `quarto add /path/to/local-repo/branch/location`
- `Y` to confirm install
- Run `quarto preview` to view "base" without theme applied
- Followed README and added `project: type: posit-docs` and `format: posit-docs-html:default`.
- Saved/preview.
- Verified that theme changes were applied.
- I also verified that the scss file was present/what I had added.

**Pull request merged**